docs/windows.md

```diff
@@ -120,153 +154,53 @@ how you can run Antrea in a Pod.
 Configure the Antrea for Linux on the control-plane Node following [Getting started](getting-started.md)
```

Comment:
"Configure the Antrea" -> "Deploy Antrea"

Reply:
Updated.